src/app/shared/components/template/components/layout/popup.ts

```diff
@@ -83,25 +88,19 @@ import { TemplateContainerComponent } from "../../template-container.component";
  * When opening a template as a popup, provide a minimal interface and load
  * the template directly as a regular template-container element
  */
-export class TemplatePopupComponent implements ITemplateContainerProps, OnInit {
-  @Input() name: string;
-  @Input() templatename: string;
-  @Input() parent?: TemplateContainerComponent;
-  @Input() row?: FlowTypes.TemplateRow;
-  /** Standalone mode shows in fullscreen and dismisses on completed/uncompleted emit events */
-  @Input() standalone?: boolean;
+export class TemplatePopupComponent {
```

Comment:
The fullscreen popup component has been both simplified and extended, so that instead of multiple configurable inputs it receives just one, but that is a json object with more options.